clients/launcher/lib/widgets/welcome_modal.dart

```diff
@@ -110,191 +188,631 @@ class _WelcomeModalContentState extends State<_WelcomeModalContent> {
+                            child: SelectableText.rich(
+                              TextSpan(
+                                children: [
+                                  TextSpan(
+                                    text: _currentWalletData['bip39_bin'] ?? '',
+                                    style: const TextStyle(
+                                      fontFamily: 'RobotoMono',
+                                      fontSize: 8,
+                                      color: Colors.white,
+                                    ),
+                                  ),
+                                  TextSpan(
+                                    text: _currentWalletData['bip39_csum'] ?? '',
+                                    style: const TextStyle(
+                                      fontFamily: 'RobotoMono',
+                                      fontSize: 8,
+                                      color: Color(0xFF4CAF50),
+                                    ),
+                                  ),
+                                  if (_currentWalletData['bip39_csum'] != null)
+                                    TextSpan(
+                                      text: '  Checksum hex: ',
+                                      style: const TextStyle(
+                                        fontFamily: 'RobotoMono',
+                                        fontSize: 8,
+                                        color: Colors.white,
+                                      ),
+                                    ),
+                                  if (_currentWalletData['bip39_csum'] != null)
+                                    TextSpan(
+                                      text: _currentWalletData['bip39_csum_hex'] ?? '',
+                                      style: const TextStyle(
+                                        fontFamily: 'RobotoMono',
+                                        fontSize: 8,
+                                        color: Color(0xFF4CAF50),
+                                      ),
+                                    ),
+                                ],
+                              ),
+                            ),
+                          ),
+                        ],
+                      ),
+                    ],
                   ),
-                  const SizedBox(width: 8),
-                  SailButton.primary(
-                    'Create Wallet',
-                    onPressed: _showAdvanced ? () => _handleCreateWallet() : () {},
-                    size: ButtonSize.regular,
+                  const SizedBox(height: 4),
+                  // HD Key Info
+                  Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      const Text(
+                        'HD Key Information',
+                        style: TextStyle(fontSize: 9, fontWeight: FontWeight.bold, color: Colors.white),
```

Comment:
There are more instances of hard-coding colors, but I'm going to stop mentioning them from here on out!